src/controllers/groupe.controller.ts

```diff
@@ -0,0 +1,83 @@
+import { NextFunction, Request, Response } from 'express';
+import { Group } from '@prisma/client';
+import { CreateGroupDto } from '@dtos/group.dto';
+import groupService from '@services/groups.service';
+
+class GroupController {
+  public groupService = new groupService();
```

Comment:
It's more conventioinal to declare it once and possibly initialize it in the constructor of the class to avoid issues with case sensitivity and naming conflicts between the calss and variable names.
```
private groupService: GroupService;
constructor() {this.groupService = new GroupService();}
```